source/_integrations/geniushub.markdown

```diff
@@ -7,24 +7,32 @@ ha_category:
   - Water Heater
   - Sensor
   - Binary Sensor
+  - Switch
 ha_release: 0.92
 ha_iot_class: Local Polling
 ---
 
-The `geniushub` integration links Home Assistant with your Genius Hub CH/DHW system, including its zones, devices, and issues.
+The `geniushub` integration links Home Assistant with your Genius Hub CH/DHW system, including its Zones, Devices, and Issues.
 
-It uses the [geniushub](https://pypi.org/project/geniushub-client/) client library, which provides data compatible with the v1 API that _may not_ necessarily match that of the official Web App.
+It uses the [geniushub](https://pypi.org/project/geniushub-client/) client library, which provides data compatible with the v1 API that _may not_ exactly match that of the official Web App.
 
 ### Zones
 
-Each zone controlled by your Genius Hub will be exposed as either a:
+Each Zone controlled by your Genius Hub will be exposed as either a:
```

Comment:
```suggestion
Each Zone controlled by your Genius Hub will be exposed as either a:
```

@zxdavb Why capitalize "Zone"? It is not a brand, but a regular word, that is not appearing in the beginning of a sentence, so it should be lower case.

Please adjust those occurrences.

Reply:
done